### PR TITLE
Use dotful DNSNames in remotebackend

### PR DESCRIPTION
**Pull request by cmouse**

### CODE REVIEW

**Comment by Habbie** [comment]
Please replace 'tailing' with 'trailing' everywhere

File: modules/remotebackend/regression-tests/test-schema.sql [hunk 1]
```diff
@@ -64,23 +64,23 @@ create table tsigkeys (
 
 create unique index namealgoindex on tsigkeys(name, algorithm);
 
-insert into domains (name,type) VALUES('example.com','NATIVE');
-insert into domains (name,type) VALUES('up.example.com','NATIVE');
+insert into domains (name,type) VALUES('example.com.','NATIVE');
+insert into domains (name,type) VALUES('up.example.com.','NATIVE');
```
Comment (Habbie) [inline modules/remotebackend/regression-tests/test-schema.sql]: Am I right in understanding that before this change, your schema and data were gsql-compatible, but this changes here?

Comment (cmouse) [inline modules/remotebackend/regression-tests/test-schema.sql]: Well I don't want to handle this in test suite code

Comment (Habbie) [inline modules/remotebackend/regression-tests/test-schema.sql]: So yes?

**Comment by cmouse** [comment]
I guess so. There never was intent to be compatible.

**Comment by cmouse** [comment]
Word tailing has been excised and replaced with trailing.

**Comment by Habbie** [comment]
LGTM

**Pull request merged**